cargo sqlx prepare --check が comment_integration_test のコンパイルエラーで失敗する

## 概要

`cargo sqlx prepare --check --workspace -- --all-targets` が `comment_integration_test.rs` のコンパイルエラーで失敗する。

## 再現手順

```bash
cd backend && cargo sqlx prepare --check --workspace -- --all-targets
```

## エラー内容

```
error[E0432]: unresolved import `ringiflow_core_service::test_utils`
  --> apps/core-service/tests/comment_integration_test.rs:6:4
   |
 6 |    test_utils::WorkflowTestBuilder,
   |    ^^^^^^^^^^ could not find `test_utils` in `ringiflow_core_service`
   |
note: found an item that was configured out
  --> backend/apps/core-service/src/lib.rs:13:9
   |
11 | #[cfg(any(test, feature = "test-utils"))]
   |          ------------------------------ the item is gated here
12 | #[doc(hidden)]
13 | pub mod test_utils;
   |         ^^^^^^^^^^
```

## 原因

`test_utils` モジュールは `#[cfg(any(test, feature = "test-utils"))]` でゲートされている。

`tests/` ディレクトリの統合テストは別クレートとしてコンパイルされるため `#[cfg(test)]` が有効にならない。`test-utils` feature を `dev-dependencies` で有効にする必要がある。

```toml
# Cargo.toml の dev-dependencies に以下が必要
[dev-dependencies]
ringiflow-core-service = { path = ".", features = ["test-utils"] }
```

## 影響

`just check-all` の `cargo sqlx prepare --check` ステップが失敗する（main ブランチでも再現）。

--- a/backend/apps/core-service/Cargo.toml
+++ b/backend/apps/core-service/Cargo.toml
@@ -25,6 +25,7 @@ dotenvy.workspace = true
 itertools.workspace = true
 
 [dev-dependencies]
+ringiflow-core-service = { path = ".", features = ["test-utils"] }
 ringiflow-domain = { workspace = true, features = ["test-support"] }
 ringiflow-infra = { workspace = true, features = ["test-utils"] }
 tokio-test.workspace = true

